iosxr_setup.py

```diff
@@ -113,60 +141,58 @@ def setup_networking(self, host_ip=None, gateway=None, allow_root_login=False):
 
         # Needed for jenkins if using root password
         if allow_root_login:
-            xr1.send("run sed -i 's/PermitRootLogin no/PermitRootLogin yes/' /etc/ssh/sshd_config_operns")
-
-        #
-        # Send commands to XR Linux
-        #
-        xr1.send("run")
+            xr1.send("bash -c sed -i 's/PermitRootLogin no/PermitRootLogin yes/' /etc/ssh/sshd_config_operns")
 
         # Add passwordless sudo as required by jenkins
-        xr1.send("echo '####Added by iosxr_setup to give vagrant passwordless access' >> /etc/sudoers")
-        xr1.send("echo '%sudo ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers")
+        # sudo not vagrant because we are operating in xrnns and global-vrf user space
```

Comment:
Not sure I understand this comment or the change back from `vagrant` to `%sudo` - can you explain?